src/DeferredFutures.jl

```diff
@@ -47,6 +47,21 @@ function Base.show(io::IO, ref::DeferredFuture)
     print(io, "$(typeof(ref).name.name) at ($(rc.where),$(rc.whence),$(rc.id))")
 end
 
+"""
+    serialize(s::AbstractSerializer, ref::DeferredFuture)
+
+Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
+"""
+function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    if VERSION < v"0.6.0-rc1"
```

Comment:
Might want to specify a specific julia version where this change (e.g., v"0.6.0-dev+xxxx") or include a link. Looks like this change was made in this PR https://github.com/JuliaLang/julia/commit/bd804ee3224d8fddb4e3d33eb053922fb33af079